src/darwin/Framework/chip_xcode_build_connector.sh

```diff
@@ -100,14 +100,35 @@ fi
     )
 }
 
+updirs() {
+    declare lookinfer="${1}"
```

Comment:
That's meant to be "looking for"?  It took me a while to figure out the naming....  Maybe call it `to_find` or some such?

Reply:
yes, "looking for"

I wrote this code years ago for my own dotfiles